### DISCUSSION TITLE
How to compile rightly from source

### DISCUSSION THREAD

**Discussion by futurist** [swiftbar/SwiftBar#360] [Q&A] [answered]
I've tried to compile from source, but with below errors:

![image](https://user-images.githubusercontent.com/159167/216652082-60da387e-fa30-4cdc-8c33-44ba52d5f35b.png)

Xcode: Version 14.2 (14C18)
macOS: 13.0.1 (22A400)

BTW, is there any more docs about how to compile from source?

**Comment by melonamin** [comment, marked as answer]
Please use SwiftBar target, not SwiftBar MAS. This should do it.
